handler-proxy/src/main/java/io/netty/handler/proxy/HttpProxyHandler.java

```diff
@@ -47,9 +47,10 @@
     private final String username;
     private final String password;
     private final CharSequence authorization;
+    private final HttpHeaders outboundHeaders;
     private final boolean ignoreDefaultPortsInConnectHostHeader;
     private HttpResponseStatus status;
-    private HttpHeaders headers;
+    private HttpHeaders inboundHeaders;
```

Comment:
It seems this should be a local variable. I don't see it referenced anywhere outside of `handleResponse()`.

Reply:
Oh, right. I remember that now. All LGTM then.